torch/onnx/symbolic_helper.py

```diff
@@ -192,6 +193,40 @@ def _slice_helper(g, input, axes, starts, ends, steps=None, dynamic_slice=False)
         from torch.onnx.symbolic_opset10 import _slice
         return _slice(g, input, axes, starts, ends, steps, dynamic_slice)
 
+
+def _sort_helper(g, input, dim, decending=True, out=None):
+    if out is not None:
+        _unimplemented("Sort", "Out parameter is not supported")
+    shape_ = g.op("Shape", input)
+    axis = g.op("Constant", value_t=torch.tensor(0, dtype=torch.int64))
+    start = g.op("Constant", value_t=torch.tensor(dim, dtype=torch.int64))
+    end = g.op("Constant", value_t=torch.tensor(dim + 1, dtype=torch.int64))
+    slice_ = _slice_helper(g, shape_, axes=axis, starts=start, ends=end, steps=None, dynamic_slice=True)
+    if _export_onnx_opset_version <= 10:
```

Comment:
Ideally my preference is to not capture the entire implementation in a helper function like this, especially when the implementation differs between two opsets like in this case. We should write the implementation in the opset's respective symbolic_opset*. We can capture the common parts of the implementation in a helper function.

Reply:
Yeah, makes sense